app/modules/individuals/resources.py

```diff
@@ -217,11 +217,58 @@ def patch(self, args, individual):
         context = api.commit_or_abort(
             db.session, default_error_message='Failed to update Individual details.'
         )
+
         with context:
             parameters.PatchIndividualDetailsParameters.perform_patch(
                 args, obj=individual
             )
-            db.session.merge(individual)
+
+        # copycat the pattern established in sightings module
+        edm_count = 0
+        for arg in args:
+            if (
+                'path' in arg
+                and arg['path'][1:]
+                in parameters.PatchIndividualDetailsParameters.PATH_CHOICES_EDM
+            ):
+                edm_count += 1
+        if edm_count > 0 and edm_count != len(args):
+            log.error(f'Mixed edm/houston patch called with args {args}')
+            abort(
+                success=False,
+                passed_message='Cannot mix EDM patch paths and houston patch paths',
+                message='Error',
+                code=400,
+            )
+
+        if edm_count > 0:
+            log.debug(f'wanting to do edm patch on args={args}')
+            result = None
+            try:
+                (
+                    response,
+                    response_data,
+                    result,
+                ) = current_app.edm.request_passthrough_parsed(
+                    'individual.data',
+                    'patch',
+                    {'data': args},
```

Comment:
Currently this will pass all args to EDM, EDM ones and houston ones (currently there are no Houston ones). 
I'm thinking that a better idea would be to have a pattern that used list comprehension for the bit at line 227 to create an edm_args and then just check the length of that.